src/main/java/com/bootcamp/demo/firebase/FirebaseController.java

```diff
@@ -30,18 +21,11 @@
 @RestController
```

Comment:
This would is for the line above, but github does not allow adding comments there.
Check if it makes sense to keep the `@ConditionalOnProperty` annotation with the current setup.